contents/docs/integrate/feature-flags-code/_snippets/override-server-properties/override-geoip-properties-intro.mdx

```diff
@@ -0,0 +1,3 @@
+### Overriding GeoIP properties
+
+By default, a user's GeoIP properties are set using the IP address they use to capture events on the frontend. You may want to override the these properties when evaluating feature flags. A common reason to do this is when you're not using PostHog on your frontend, so the user has no GeoIP properties. 
```

Comment:
What about backend? I forget off the top of my head if we drop GeoIP properties for backend SDKs by default or use server IP. Would be good to clarify this here.

Reply:
My understanding from [this conversation](https://github.com/PostHog/posthog.com/pull/6629#discussion_r1318186671) is that we dont use the geoIPs from the backend, since then users IP would just be your servers, which doesnt make sense.